index.html

```diff
@@ -11,7 +11,7 @@
 
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="style.css">
```

Comment:
- [ ] There is an issue with your style.css file, the style is not applicated to your Html element, kindly fix that

Reply:
Resolved